src/com/lightdev/app/shtm/resources/SimplyHTML.properties

```diff
@@ -408,7 +407,7 @@ docNameMissingError=Please specify name and location for the document through 'S
 unableToUndoError=Unable to undo:
 unableToRedoError=Unable to redo:
 unableToOpenFileError=File can not be opened
-cantCreateURLError=Can not create a valid URL for\n
+cantCreateURLError=Can not create a valid URL for
```

Comment:
Are you sure \n is a mistake?

Reply:
sorry, you're right, reverting this change